src/installation/live-images/index.md

```diff
@@ -1,55 +1,24 @@
-| Document Viewer   | -                                                     | -               | -                                       | -              | Atril (PS/PDF)                                                                                                                                                      | -                                                                                                                                   |
-| Plain text viewer | -                                                     | -               | -                                       | -              | Pluma                                                                                                                                                               | Mousepad                                                                                                                            |
-| Image viewer      | -                                                     | -               | GPicView                                | LXImage        | Eye of MATE                                                                                                                                                         | Ristretto                                                                                                                           |
-| Archive unpacker  | -                                                     | -               | -                                       | -              | Engrampa                                                                                                                                                            | -                                                                                                                                   |
-| Other             | Mixer, EConnMan (connection manager), Elementary Test | -               | LXTask (task manager), MIME type editor | Screen grabber | Screen grabber, file finder, MATE color picker, MATE font viewer, Disk usage analyzer, Power statistics, System monitor (task manager), Dictionary, Log file viewer | Bulk rename, Orage Globaltime, Orage Calendar, Task Manager, Parole Media Player, Audio Mixer, MIME type editor, Application finder |
+Void provides live images containing a base set of utilities useful for
+installing and/or repairing systems. Images can be found on the
+[mirrors](../../xbps/repositories/mirrors/index.md) for either `glibc` or `musl`
+systems on `x86_64` systems, and for `glibc` on `i686` ones. Users of other
+architectures will need to perform a [manual installation](../guides/index.md).
+
+## Base Image
+
+The base image contains just enough for a usable Void system, with utilities for
```

Comment:
```suggestion
The base image contains just enough tools for installing a usable Void system, with utilities for
```

Reply:
I think "includes just enough tools to install a usable Void system" works even better?